### ISSUE TITLE
[Bug] No Good Training Graph

### ISSUE THREAD

**Issue by yehyunsuh** [yehyunsuh/Total-Knee-Replacement-Post-Surgical-Assessment#9]
## What
Strange training graph
![image](https://user-images.githubusercontent.com/73840274/209410732-acbaa080-9f8c-4458-a9a8-fe5c87762703.png)

## Why
Do not know -> will try to first overfit the model 

## How
- [x] Full batch size
- [x] Eliminate batch normalization
- [x] Dilate pixels
- [x] Spatial mean + MSE Loss should show a good graph

**Comment by yehyunsuh** [comment]
Did not show a good training process with the change. Close issue.
